src/qutip_tensornetwork/core/data/network.py

```diff
@@ -0,0 +1,627 @@
+              len(nodes)))
+        array = list(nodes)[0].tensor
+
+        return array.reshape(self.shape)
+
+    @classmethod
+    def from_2d_array(cls, array):
+        """Create a network from a 2D, 1D or scalar array. This network will
+        have a single node with out_edges referring to the first index of the
+        array and in_edges referring to the second index of the array. If any of
+        those has dimension 1, it is ignored when creating in_edges or
+        out_edges.
+
+        Parameters
+        ----------
+        array: ndarray or Data
+            Array from which the single node of the network is created.
+
+        Returns
+        -------
+        network: Network
+            Instance of Network with a single node representing array.
+
+        Examples
+        --------
+        >>> array = np.array((2, 2))
+        >>> net = Network.from_2d_array(array)
+        >>> net.dims
+        [[2], [2]]
+
+        One dimensional arrays are understoof as kets and cero dimensional ones
+        as scalars.
+
+        >>> array = np.array((2)) # ket
+        >>> net = Network.from_2d_array(array)
+        >>> net.dims
+        [[2], []]
+
+        If the array has one dimension being 1, it is reshaped before the
+        Network instantiation.
+
+        >>> array = np.array((2, 1)) # ket
+        >>> net = Network.from_2d_array(array)
+        >>> net.dims
+        [[2], []]
+        """
+
+        if isinstance(array, qutip.data.Data):
+            array = array.to_array()
+        elif not isinstance(array, np.ndarray):
+            raise ValueError("`array` is not instance of Data or np.array.")
+
+        shape = array.shape
+        if len(shape) > 2:
+            raise ValueError("This method only works with 2D, 1D or scalar "
+                             "arrays but the"
+                             "input has " + str(len(array.shape)) + "dimensions")
+
+        if len(shape)==1:
+            node = tn.Node(array)
+            return Network(node[:], [])
+
+        if len(shape)==0:
+            node = tn.Node(array)
+            return Network([], [], [node])
+
+        if array.shape[0]==1 and array.shape[1]!=1:
+            array = array.reshape(array.shape[1])
+            node = tn.Node(array)
+            return Network([], node[:])
+
+        elif array.shape[0]!=1 and array.shape[1]==1:
+            array = array.reshape(array.shape[0])
+            node = tn.Node(array)
+            return Network(node[:], [])
+
+        elif array.shape[0]==1 and array.shape[1]==1:
+            array = array.reshape(())
+            node = tn.Node(array)
+            return Network([], [], nodes=[node])
+
+        else:
+            node = tn.Node(array)
+            return Network(node[0:1], node[1:])
+
+    def as_quoperator(self):
+        # TODO: This can not be implemented unless tensornetwork includes the 
+        # __init__ file for the quantum module.
+        raise NotImplementedError()
```

Comment:
Let's just delete this for now, I think -- I can't see a reason for us to make a quoperator right now.

Reply:
I agree.